core/templates/components/forms/schema-based-editors/schema-based-list-editor.component.html

```diff
@@ -59,3 +59,10 @@
     </div>
   </form>
 </div>
+<style>
+  @media screen and (max-width: 691px) {
```

Comment:
Should we include this style along with the other styles on the top of this HTML?